bob-apps/bin/bobd.rs

```diff
@@ -191,5 +235,12 @@ fn get_matches<'a>() -> ArgMatches<'a> {
                 .long("port")
                 .takes_value(true),
         )
+        .arg(
+            Arg::with_name("init folders")
+                .help("Initializes bob and alien folders")
+                .short("i")
```

Comment:
This is a special one-time flag, so I don't see any benefit in using a short name. It's better to use only the long name to improve command readability